ninja.py

```diff
@@ -248,14 +248,15 @@ def ninja_filter(inputSeqsFile, filteredSeqsFile, seqsDBFile, trim, RC, denoisin
     cmd = ""
     try:
         global shellBool
-        cmd = ninjaFilterFile + ' ' + inputSeqsFile + ' ' + filteredSeqsFile + ' ' + seqsDBFile + ' ' + argTrim + \
-              ' ' + argRC + ' ' + argDenoising
+        # cmd = ninjaFilterFile + ' ' + inputSeqsFile + ' ' + filteredSeqsFile + ' ' + seqsDBFile + ' ' + argTrim + \
+            #   ' ' + argRC + ' ' + argDenoising
+        cmd = [ninjaFilterFile, inputSeqsFile, filteredSeqsFile, seqsDBFile, argTrim, argRC, argDenoising]
         print cmd
-        subprocess.check_call(cmd, shell = shellBool, stdout = sys.stdout)
+        subprocess.check_call(cmd, stdout = sys.stdout)

```

Comment:
This call should be cross platform. The use of shell should never happen as it is a security risk. This also leads to the bug with spaces in directory paths.